pages/12.Plugins/22 GrapesJs/docs.en.md

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
As with the other file - I think this page is not required. Let's remove the whole folder and keep everything under the Builder section.